src/modules/admin/components/Tokens/TokensMoveDialogForm.tsx

```diff
@@ -0,0 +1,320 @@
+      values.toDomain !== undefined &&
+      values.toDomain === values.fromDomain
+    ) {
+      setErrors({ toDomain: MSG.samePot });
+    }
+  }, [
+    fromDomainPermissions,
+    fromDomainTokenBalance,
+    isFetchingFromDomainPermissions,
+    isFetchingToDomainPermissions,
+    selectedTokenRef,
+    setErrors,
+    toDomainPermissions,
+    values.amount,
+    values.fromDomain,
+    values.toDomain,
+  ]);
+
+  return (
+    <>
+      <FormStatus status={status} />
+      <DialogSection>
+        <Heading
+          appearance={{ size: 'medium', margin: 'none' }}
+          text={MSG.title}
+        />
+      </DialogSection>
+      <DialogSection>
+        <Select options={domainOptions} label={MSG.from} name="fromDomain" />
+        {values.fromDomain !== undefined && !!values.tokenAddress && (
+          <div className={styles.domainPotBalance}>
+            <FormattedMessage
+              {...MSG.domainTokenAmount}
+              values={{
+                amount: (
+                  <Numeral
+                    appearance={{
+                      size: 'small',
+                      theme: 'grey',
+                    }}
+                    value={fromDomainTokenBalance || 0}
+                    unit={(selectedToken && selectedToken.decimals) || 18}
+                    truncate={3}
+                  />
+                ),
+                symbol: selectedToken.symbol || '???',
+              }}
+            />
+          </div>
+        )}
+      </DialogSection>
+      <DialogSection>
+        <Select options={domainOptions} label={MSG.to} name="toDomain" />
+        {values.toDomain !== undefined && !!values.tokenAddress && (
+          <div className={styles.domainPotBalance}>
+            <FormattedMessage
+              {...MSG.domainTokenAmount}
+              values={{
+                amount: (
+                  <Numeral
+                    appearance={{
+                      size: 'small',
+                      theme: 'grey',
+                    }}
+                    value={toDomainTokenBalance || 0}
+                    unit={(selectedToken && selectedToken.decimals) || 18}
+                    truncate={3}
+                  />
+                ),
+                symbol: selectedToken.symbol || '???',
+              }}
+            />
+          </div>
+        )}
+      </DialogSection>
+      <DialogSection>
+        <div className={styles.tokenAmount}>
+          <div>
+            <Input
+              label={MSG.amount}
+              name="amount"
+              appearance={{ theme: 'minimal', align: 'right' }}
+              formattingOptions={{
+                delimiter: ',',
+                numeral: true,
+                numeralDecimalScale:
+                  (selectedToken && selectedToken.decimals) || 18,
+              }}
+            />
+          </div>
+          <div className={styles.tokenAmountSelect}>
+            <Select options={tokenOptions} name="tokenAddress" elementOnly />
```

Comment:
The select component should be made to open to the left. As it is, it adds a bottom scroll-bar, plus it cuts the options list:

![Screenshot from 2019-09-25 09-19-57](https://user-images.githubusercontent.com/1193222/65575336-3fbe0c80-df78-11e9-9477-fe7a95d995a5.png)